File: mario-more/mario.c
#include <stdio.h>
#include <cs50.h>

int main(void)
{
    int n = get_int("Height: ");
    for(int i=n; i > 0; i--)
        {
        for(int j=(n-1); j > 0; j--)
            {
                printf(" ");
            }
        for(int k=1; k <= n ; k++)
            {
                printf("#");
            }
        printf(" \n");
        }
}

File: python_/sentimental-mario-more/mario.py
import cs50

height = 0
# Loop, continuing to ask for a heigh if not between 1 - 8
while True:
    height = cs50.get_int("How tall do you want it?: ")
    if 9 > height > 0:
        break

# Print the blocks
for i in range(height):
    print((height - 1 - i) * " " + i * "#" + "#  #" + i * "#")